CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Internal refactor of Web API functionality.
+- `Geometry.from_gds` doesn't create unecessary groups of single elements.
```

Comment:
slight tweak, maybe just 
```
`Geometry.from_gds` returns `Geometry` instead of `GeometryGroup` for single elements.
```

Reply:
I think probably this comment is also irrelevant because I forgot that GeometryGroup is a Geometry type.